src/Paramore.Brighter/CommandProcessor.cs

```diff
@@ -508,11 +582,23 @@ public async Task PostAsync<T>(T request, bool continueOnCapturedContext = false
 
             var message = messageMapper.MapToMessage(request);
 
+            AddCloudEventsToMessage<T>(message);
+
             await _bus.AddToOutboxAsync(request, continueOnCapturedContext, cancellationToken, message, connectionProvider);
 
             return message.Id;
         }
 
+        private void AddCloudEventsToMessage<T>(Message message)
```

Comment:
See earlier comments. I don't think we know in the CP whether you are using CE or not. That lives with the transport (and possibly enveloper wrapper later). All this should know is that MessageHeader has some appropriate properties

Reply:
I think this is terribly named on my part. I think it's should be called something closer to AddTelemetryToMessage or EnsureMessageTelemetry